### PR TITLE
Chore: fix typos, rename site

### PR DESCRIPTION
**Pull request by esune**
Fixed a few typos and renamed the main site to `Common Hosted Digital Credential Service`

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -21,5 +21,5 @@ There are three main patterns that can be leveraged in the Digital Trust space,
...
-- [Traction](solutions/traction-overview.md): a Software-As-A-Service Enterprise agent service based on [Aries Cloudagent Python](https://github.com/hyperledger/aries-cloudagent-python), it provides streamlined onboarding for new adopters wanting to have full control over their Digital trust processes, from receiving and presenting Digital credentials to acting as a verifier or even an issuer.
+- [Traction](solutions/traction-overview.md): a Software-As-A-Service Enterprise agent service based on [Aries Cloudagent Python](https://github.com/hyperledger/aries-cloudagent-python), it provides streamlined onboarding for new adopters wanting to have full control over their Digital Trust processes, from receiving and presenting Digital credentials to acting as a verifier or even an issuer.
```
Comment (loneil) [inline docs/index.md:24]: There's the hyperledger > OWF renamings as well, not sure if this is the only place but could do those too...

"Aries Cloudagent Python" -> "ACA-Py"

hyperledger -> OWF

File: docs/acapy-integration-howto.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Integration How-To
 
-Implementing a custom integration with an ACA-Py/Traction agent is fairly straightforward: the agent exposes a set of REST APIs that cover all of the Digital trust functionality that may be implemented. There are, however, some concepts that need to be considered when making requests to these endpoints. This page outlines some of the most important concepts and "gotchas" that need to be considered.
+Implementing a custom integration with an ACA-Py/Traction agent is fairly straightforward: the agent exposes a set of REST APIs that cover all of the CommonDigital Credential Service functionality that may be implemented. There are, however, some concepts that need to be considered when making requests to these endpoints. This page outlines some of the most important concepts and "gotchas" that need to be considered.
```
Comment (loneil) [inline docs/acapy-integration-howto.md:3]: "CommonDigital"

**Review by loneil — COMMENTED** [review]
One typo and other note

File: docs/index.md (hunk 1 above)
Comment (esune) [reply docs/index.md:24]: Oh, good catch! On it.

**Pull request merged**